notebook/Tutorial_VertexCover.ipynb

```diff
@@ -0,0 +1,897 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sampleset = qaoa_converter.decode(qk_transpiler, job_result.data[\"meas\"])\n",
```

Comment:
Can you change the plot code like quickstart (https://jij-inc.github.io/Qamomile/quickstart.html#decode-the-result)
The reason is in https://github.com/Jij-Inc/Qamomile/issues/76#issue-2562951830